js/webapp.js

```diff
@@ -161,6 +160,17 @@ const staticHandlers = {
         });
     });
     return r;
+  },
+  
+  eureka() {
+    const router = express.Router();
+    router.get('/application/info', function(req, res, next) {
```

Comment:
It's flexible: it's actually specified by us when registering to the APIML, see here: https://github.com/zowe/zlux-proxy-server/blob/fa8cd1b5c6b63033aa33f81b9f3951aa1e7d5a72/js/apiml.js#L92

Reply:
Then, can it be changed to:
```
statusPageUrl: `${proto}://${this.hostName}:${port}/server/apiml/info`,
healthCheckUrl: `${proto}://${this.hostName}:${port}/server/apiml/health`,
```
Because, these are very specific pages, and /application will be confused since this is an application server.

Reply:
Well, I wouldn't have 'apiml' in those URIs though, because it's *our* health, not APIML's.